cmd/run.go

```diff
@@ -38,13 +38,13 @@ func addRunSubcommands() {
 }
 
 func createSingleRunCmd(checkerType okgo.CheckerType, factory okgo.CheckerFactory) *cobra.Command {
-	checker, err := factory.NewChecker(checkerType, nil)
-	if err != nil {
-		panic(errors.Wrapf(err, "failed to create command for checker type %s", checkerType))
-	}
 	return &cobra.Command{
 		Use: string(checkerType),
 		Run: func(cmd *cobra.Command, args []string) {
+			checker, err := factory.NewChecker(checkerType, nil)
+			if err != nil {
+				panic(errors.Wrapf(err, "failed to create command for checker type %s", checkerType))
```

Comment:
If we keep this change, then we should assign `RunE` instead of `Run` and then return the error instead of calling `panic` -- the `panic` made sense in `createSingleRunCmd` since that happens when constructing the CLI itself so there's no other good error mechanism to speak of, but if we're doing it at command runtime we should use the regular CLI error mechanisms rather than `panic`